test-network-function/accesscontrol/suite.go

```diff
@@ -130,16 +156,103 @@ func runTestOnPods(env *config.TestEnvironment, testCmd testcases.BaseTestCase,
 	})
 }
 
+func getCrsNamespaces(crdName, crdKind string) map[string]string {
+	gomega.Expect(crdKind).NotTo(gomega.BeEmpty())
+	getCrNamespaceCommand := fmt.Sprintf(ocGetCrNamespaceFormat, crdKind)
+	cmdOut := utils.ExecuteCommand(getCrNamespaceCommand, common.DefaultTimeout, common.GetContext(), func() {
+		tcClaimLogPrintf("CRD %s: Failed to get CRs (kind=%s)", crdName, crdKind)
+	})
+	// No CRs created for this CRD yet.
+	if cmdOut == "" {
+		return map[string]string{}
+	}
+	lines := strings.Split(cmdOut, "\n")
+
+	crNamespaces := map[string]string{}
+	for _, line := range lines {
+		if line == "" {
+			// This is to filter the last line, which is always empty.
+			continue
+		}
+		lineFields := strings.Split(line, ",")
+		crNamespaces[lineFields[0]] = lineFields[1]
```

Comment:
is it safe to assume lineFields will always have length ==2 ? I would prefer having a check for safety

Reply:
I've used a custom go-template for the oc get command so the output should always be "crName,namespace" per line, but I agree a safety check is a always a good idea.